src/plugins/camelCase.js

```diff
@@ -0,0 +1,2 @@
+import { camelizeKeys } from 'humps';
+export default data => camelizeKeys(data);
```

Comment:
I guess you can just export camlizeKeys directly?
`export default camelizekeys;`

Reply:
Yeah but I like the separation between the used package and the implemented function. Definitely when in the future another package is used. (Just my opninion though)

Reply:
I think he just meant

```
import { camelizeKeys } from 'humps';
export camelizeKeys;

```

@lowiebenoot ?

Reply:
Yes (but `export default`)

Reply:
exactly 🤐